lib/rules/no-warning-comments.js

```diff
@@ -141,12 +143,28 @@ module.exports = {
 
             const matches = commentContainsWarningTerm(node.value);
 
-            matches.forEach(matchedTerm => {
+            matches.forEach(({ type, comment }) => {
+                let len = 0;
+                const comments = [];
+                let truncated = false;
+
+                comment
+                    .split(/\s/u).forEach(c => {
+                        if (c.length + len <= CHAR_LIMIT) {
+                            comments.push(c);
+                        } else {
+                            truncated = true;
+                        }
+
+                        len += c.length;
```

Comment:
Should we also count spaces that will be added between?

Reply:
Actually, I am counting only the readable char. 

We can count the number of space, (`comments.lenght - 1`)